examples/solvers/adupdates_tomography.py

```diff
@@ -0,0 +1,149 @@
+r"""Total-variation regularized tomography example using the adupdates solver.
+
+This example solves a linear inverse problem of the form :math:`Ax = y`, where
+:math:`x \in \mathbb{R}^n` is the (unknown) original image to be reconstructed,
+:math:`y \in \mathbb{R}^m` is the noisy data and :math:`A \in \mathbb{R}^{m
+\times n}` is the measurement matrix describing the discretized physical model.
+
+To solve this problem, we first split the measurement matrix and the data into
+:math:`m` parts and solve the least squares problem
+
+..math::
+    \min_{x\in\mathbb{R}^n} D(x) := \sum_{j = 1}^m \| A_j x - y_j \|^2.
+
+To regularize, we add terms for the total variation and for a functional which
+guarantees that the solution will be pointwise nonnegative. The variational
+regularization problem we are solving is therefore
+
+..math::
+    \min_{x\in \mathbb{R}^n_+} D(x) + \sum_{i = 1}^d (\| \partial_{i, 1} x \|_1
+    + \| \partial_{i, 2} x \|_1.
+
+Here, :math:`\partial_{i, 1}` and :math:`\partial_{i, 2}` contain the even and
+odd components, respectively, of the discretized :math:`i`th partial
```

Comment:
Not sure, but
```
:math:`i`th
```
may not be rendered correctly -- the backtick must be followed by a whitespace or punctuation, as far as I know.
```
:math:`i`-th
```
should work, though.